app/model/commands/ActiveAssetCommand.scala

```diff
@@ -58,8 +59,10 @@ case class ActiveAssetCommand(atomId: String, youtubeId: String)
                 duration = ytAssetDuration
               ))
 
-            UpdateAtomCommand(atomId, MediaAtom.fromThrift(updatedAtom)).process()
-            PublishAtomCommand(atomId).process()
+            publishedDataStore.getAtom(atomId) match {
```

Comment:
"Pressing the publish button is the only way to publish an atom: atom will no longer be published when an atom is added to it"

Does that cover this code as well? My naive reading of it is that we re-publish an already published atom if the asset is changed